apps/web/pages/index.tsx

```diff
@@ -1,16 +1,44 @@
-import { Center, Heading } from '@chakra-ui/react';
+import { Box, Divider, Flex, Heading, Text } from '@chakra-ui/react';
 import type { NextPage } from 'next';
 import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { Topbar, BottomBar } from '../components/Navbar';
```

Comment:
The issue with this consider the layout changes or another component is needed. WE need to do this everywhere.

This is where layout component comes into picture. Use that wherever layout is needed

Ref: https://adamwathan.me/2019/10/17/persistent-layout-patterns-in-nextjs/

Reply:
the topbar have a option to render a button or not and it depends on routes thats y renderd the component this way 
eg ->
  in some page button is logout,login,etc

and yes i totally forgot about rendering the bottombar in "_app.tsx" sorry about that

Reply:
Move all these logic into a layout component and use the strategy to integrate layout as shared above.

Soon nextjs will release a feature for layouts and we will migrate  to that